### ISSUE TITLE
My melee won't work when my attack stat is too high

### ISSUE THREAD

**Issue by branman5949** [Hedaox/ninjinknockback#3]
melee won't work and i cant do melee damage to the enemy if my melee stat is above 23 million and i was wondering if you could fix that.

**Comment by Hedaox** [comment]
Yup I will need to take a look at that, but for the time being you should not use big numbers with this mod.
